src/services/user.go

```
@@ -66,6 +66,41 @@ func (service userService) Info(ctx context.Context, request domainUser.InfoRequ
+	err = validations.ValidateUserCheck(ctx, request)
+	if err != nil {
+		return response, err
+	}
+	
+	var jids []types.JID
+	dataWaRecipient, err := whatsapp.ValidateJidWithLogin(service.WaCli, request.Phone)
+	if err != nil {
+		return response, err
+	}
+
+	jids = append(jids, dataWaRecipient)
```

golangci-lint (1.62.2)

ineffectual assignment to jids

(ineffassign)